test/selected-attribute.html

```diff
@@ -66,6 +76,24 @@
 
     });
 
+    suite('changing attrForSelected', function() {
+
+      var s;
+
+      setup(function () {
+        s = fixture('test-attr-change');
+      });
+
+      test('changing selectedAttribute', function() {
+        s.attrForSelected = 'data-y';
+        s.async(function() {
```

Comment:
I tried w/o in the first place, but it seems the value is not updated properly "in time" in this case... Do you have a better solution? I had also such syncing issue while developing an iron-swipeable-pages element where selection or change of class were not propagating in time for the code to execute properly w/o some call to async()...